### DISCUSSION TITLE
Faculdade EAD ou Presencial?

### DISCUSSION THREAD

**Discussion by Raeski** [frontendbr/forum#1768] [Discussão]
Hoje em dia curso ADS ead na Estácio, estou no terceiro período porém até agora não tive nenhum aprendizado importante, as aulas são extremamente defasadas, tudo que aprendi foram com cursos por fora. Vou me mudar da cidade onde moro hoje e estava pensando em transferir meu curso para PUCMG no curso de Sistema de informação, porém os valores de um curso para outro são bem distantes, será que vale a pena fazer essa troca? Ou pelo menos trocar por uma faculdade presencial? Agradeço desde já.

**Comment by giovannism20** [comment]
> Bom, baseado no que tu escreveu entendi que você faz a distância. 

- Antes mesmo de avaliar a questão presencial ou a distância, a própria grade dos cursos é relativamente distante, é necessário verificar qual o teu objetivo com a faculdade e verificar se o curso encaixa com isso. 

- Sobre a grade da Estácio estar defasada, seria interessante tu descrever o motivo, dito que as vezes aquilo está sendo ensinado com um propósito específico, muita gente não gosta da parte de planejamento de software: diagramas, elaboração de documentos...mas só porquê parece que tu nunca vai usar aquilo (é bom lembrar que normalmente são pessoas mais experientes e foi colocado na grade com um objetivo).

> Lembre-se também que muita gente que está ali, nunca tocou em nada, demoram mais para aprender e as vezes não possuem condições de comprar um curso externo, tempo para ver mesmo os gratuitos. E, defasado ≠ consolidado Ex: Já vi falarem que Java é defasado e o certo seria ensinar JavaScript na faculdade porque é usado em mercado. A faculdade não está ali para te ensinar linguagens de programação.

**Comment by kivervinicius** [comment]
@Raeski PUC tem mais nome e alem disso é presencial pelo que eu entendi, não se prenda a conteúdo (defasado ou não) na faculdade, se prenda sempre ao aprendizado em geral, lógica, estatística, analise entre outros recursos, nunca nenhuma faculdade estará tão atualizada quanto ao mercado até por isso temos cursos adicionais. Já fui professor universitário e é muito dificil ensinar o que está na crista da onda para alunos que vieram de outros cursos ou professores onde estes não possuem a base da programação, então sempre estude a base de tudo e depois foque em linguagens ou frameworks.

**Comment by vagnerlandio** [comment]
Ano passado terminei o mesmo que curso EAD que você está fazendo e na mesma faculdade também. Não me identifiquei com a modalidade EAD mas no meu caso não tinha outra escolha. Pelo que percebi durante todo o curso é que ele é bem completo na questão de mostrar para o aluno qual caminho seguir, "e estudando outros conteúdos em paralelo" com o curso o aluno já pode sair da faculdade pronto para o mercado de trabalho.

Confesso que apesar de achar o curso bem completo, a Estácio pecou muito na metodologia usada para passar esse conhecimento para os alunos, talvez não seja culpa dela mas do ensino "tradicional" em geral.

O curso vai te mostrar que existe vários caminhos para seguir como programador, você verá C#, Java, C++, JavaScript, desenvolvimento de sistemas para desktop, web e mobile. Mas todo esse conteúdo mostrando somente a introdução e cabendo ao aluno focar em escolher uma dessas áreas e aprofundar nos estudos.

Na minha opinião, se for trocar de faculdade para continuar na modalidade EAD não vale apena pois terá quase a mesma metodologia de ensino. Já sobre a presencial, no meu caso se tivesse condições teria feito sem dúvida alguma pois muitas vezes me senti desmotivado durante o curso por não ter encontros pessoais com os outros alunos e nem mesmo trocar conhecimento com eles. Se também tiver se sentindo assim, recomendo fazer um curso presencial. No meu caso concluí o curso EAD mas pretendo fazer um pós presencial.
